mmv1/templates/terraform/examples/resource_manager_capability.tf.tmpl

```diff
@@ -0,0 +1,16 @@
+resource "google_folder" "folder" {
+  provider         = google-beta
+  display_name     = "{{index $.Vars "display_name"}}"
+  parent           = "{{index $.TestEnvVars "org_id"}}"
```

Comment:
I believe parent needs to be prefixed with "organizations/". Check other places where we set up folders for tests.